modules/microshift-nw-ipv6-single-stack-config.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-ipv6-single-stack-config_{context}"]
+= Configuring IPv6 single-stack
+
+You can use the IPv6 network protocol by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+* The host has an IPv6 address and IPv6 routes, including the default.
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+. Keep the new {microshift-short} `config.yaml` in the `/etc/microshift/` directory. Your `config.yaml` file is read every time the {microshift-short} service starts.
++
+[NOTE]
+====
+After you create it, the `config.yaml` file takes precedence over built-in settings.
+====
+
+. Replace the default values in the `network` section of the {microshift-short} YAML with your desired valid values.
++
+.Example single-stack `IPv6` configuration
+[source,yaml]
+----
+apiServer:
+# ...
+network:
+  clusterNetwork:
+  - fd01::/48 <1>
+  serviceNetwork:
+  - fd02::/112 <2>
+node:
+  nodeIP: 2600:1f14:1c48:ee00:2d76:3190:5bc2:5aef <3>
```

Comment:
Might be worth mentioning that this is only required in systems where IPv4 is also present. If the host is IPv6 only then this is not needed as MicroShift is able to automatically fill it in by itself.